src/passes/analysis/verilog/inline_utils.cpp

```diff
@@ -346,3 +354,13 @@ std::unique_ptr<vAST::Module> AlwaysStarMerger::visit(
   node->body = std::move(new_body);
   return node;
 }
+
+bool isInlined(CoreIR::Module* module, bool _inline) {
```

Comment:
is this used?

Reply:
Here: https://github.com/rdaly525/coreir/pull/928/files#diff-6c4fb4c3cc23c2c09bda419439646981R275 (I think it's hidden in the default diff since it's large)

Reply:
oh totally missed that. let me review that file